lib/commands/recordscreen.js

```diff
@@ -0,0 +1,161 @@
+import _ from 'lodash';
+import { sleep } from 'asyncbox';
+import helpers from '../android-helpers';
+
+let commands = {}, extensions = {};
+
+/**
+* record the display of devices running Android 4.4 (API level 19) and higher.
+* It records screen activity to an MPEG-4 file. Audio is not recorded with the video file.
+*
+* @param filePath the video file name
+*                 for example, "/sdcard/demo.mp4"
+* @param videoSize the format is widthxheight.
+*                  if it is "default", the default value is the device's native display resolution (if supported),
+*                  1280x720 if not. For best results,
+*                  use a size supported by your device's Advanced Video Coding (AVC) encoder.
+*                  for example, "1280x720"
+* @param timeLimit the maximum recording time, in seconds. if it is -1, the default and maximum value is 180 (3 minutes).
+* @param bitRate the video bit rate for the video, in megabits per second.
+*                if it is -1, the default value is 4Mbps. You can increase the bit rate to improve video quality,
+*                but doing so results in larger movie files.
+*                for example, 6000000
+*
+*/
+commands.startRecordingScreen = async function ( filePath, videoSize, timeLimit, bitRate ) {
+  let cmd, data, length, indexNoSuchFile;
+  let loop = true;
+  let opts = {};
+  let {udid, emPort} = await helpers.getDeviceInfoFromCaps(opts);
+  
+  opts.udid = udid;
+  opts.emPort = emPort;
+
+  let adb = await helpers.createADB(opts.javaVersion,
```

Comment:
There should already be an instance of `ADB` on the driver object. It is `this.adb`.